include/wx/gtk/dirdlg.h

```diff
@@ -55,7 +56,7 @@ class WXDLLIMPEXP_CORE wxDirDialog : public wxDirDialogBase
 
 
 private:
-    wxString m_selectedDirectory;
+    wxArrayString m_paths;
```

Comment:
is `wxArrayString` still alive! I thought it is better replaced with `wxVector<wxString>`.